### ISSUE TITLE
[News] Featured image are broken in newslist and in news detail page

### ISSUE THREAD

**Issue by srenault-meeds** [Meeds-io/meeds#2611]
**Steps to reproduce**

1. Add an image to your news
2. Publish a news to a newslist
3. See errors:
  - featured image is broken on newslist
![image](https://github.com/user-attachments/assets/feed42a8-bbd7-4c8e-a0a2-44fea0f28c13)

  - image is not displayed in the news detail page
![image](https://github.com/user-attachments/assets/8c9d00e8-07dd-4cd1-8e95-49912e9777c4)

**Expected behavior**

Images are displayed as expected

**Comment by srenault-meeds** [comment]
@Jihed525, I have reported the issue last week but as it occurs too on meeds platform, I reported the issue.

CC @azayati @Julien-Dubois-eXo

**Comment by azayati** [comment]
> @Jihed525, I have reported the issue last week but as it occurs too on meeds platform, I reported the issue.
> 
> CC @azayati @Julien-Dubois-eXo

@srenault-meeds The issue is already [reported ](https://community.exoplatform.com/portal/dw/tasks/taskDetail/75508)by @Julien-Dubois-eXo cc @rdenarie

**Comment by srenault-meeds** [comment]
Where? Because my issue reported last week on 11/13 is still on "To qualify" status

**Comment by azayati** [comment]
> Where? Because my issue reported last week on 11/13 is still on "To qualify" status

https://community.exoplatform.com/portal/dw/tasks/taskDetail/75508

**Comment by srenault-meeds** [comment]
Not exhaustive. Indeed, the featured image is not displayed in the news detail.
So you plan to merge it to develop please?

**Comment by azayati** [comment]
> Not exhaustive. Indeed, the featured image is not displayed in the news detail. So you plan to merge it to develop please?

yes it should be merged in develop as an issue on builders, @Julien-Dubois-eXo to complete the task
